gradle-baseline-java-config/resources/checkstyle/checkstyle.xml

```diff
@@ -130,6 +140,7 @@
             <property name="lineWrappingIndentation" value="8"/>
         </module>
         <module name="InnerAssignment"/> <!-- Java Coding Guidelines: Inner assignments: Not used -->
+        <module name="InterfaceIsType"/>
```

Comment:
This seems like a case where people who don't want to do this would never do it by accident, and anyone who might want to do it is not going to be persuaded otherwise because baseline gets upset. Essentially I'm not sure it adds anything, but also has the potential to annoy people.